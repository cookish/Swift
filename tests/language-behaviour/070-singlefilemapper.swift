type messagefile;

(messagefile t) write() { 
    app {
        echo @filename(t) stdout=@filename(t);
    }
}

messagefile outfile <"070-singlefilemapper.out">;

outfile = write();

