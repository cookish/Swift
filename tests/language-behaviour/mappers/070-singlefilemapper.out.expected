#!/bin/bash
set -x
cat 070-singlefilemapper.out | grep 070-singlefilemapper.out || exit 1
exit 0
